plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -202,7 +202,7 @@ export async function createRouter(
   const actionRegistry = new TemplateActionRegistry();
 
   const workers = [];
-  for (let i = 0; i < (taskWorkers || 3); i++) {
+  for (let i = 0; i < (taskWorkers || 1); i++) {
```

Comment:
We probably want to deprecate this option in favour of `concurrentTasksLimit`

Reply:
Thanks @jhaals ... that's true. Since we are on this.. if we are only creating a single worker, do we even need the `TaskWorker` or can we just use the `TaskBroker` to orchestrate the WorkflowRunners directly?

Reply:
I removed the workers array, and the router is always just creating a single worker.

Reply:
I think that we probably want to keep this as 3 right? Otherwise isn't this going to degrade performance without switching to the new property?

Reply:
Or maybe it doesn't actually matter as it's going to use the other method anyway.